quicknes/bizinterface.cpp

```diff
@@ -50,9 +50,32 @@ QN_EXPORT const char *qn_set_sample_rate(quickerNES::Emu *e, int rate)
 	return ret;
 }
 
-QN_EXPORT const char *qn_emulate_frame(quickerNES::Emu *e, int pad1, int pad2)
+
+QN_EXPORT const char *qn_emulate_frame(quickerNES::Emu *e, int pad1, int pad2, int arkanoidPosition, int arkanoidFire, int controllerType)

```

Comment:
I feel like instead of casting every single value in the function below to its real type, you could just be passing in the correct type here, e.g. pad1 as uint32_t or arkanoidFire as uint8_t.

Reply:
done